src/components/Layout.tsx

```diff
@@ -60,10 +60,39 @@ const Main = styled.main`
   flex-grow: 1;
 `
 
-const Layout = (props) => {
-  const [isDarkTheme, setIsDarkTheme] = useState(false)
-  const [isZenMode, setIsZenMode] = useState(false)
-  const [shouldShowSideNav, setShouldShowSideNav] = useState(false)
+interface PageContext {
```

Comment:
There is already a `Context` defined in `src/types`. Lets use that one here and update it if needed.